src/components/change-authenticator-app/tests/change-authenticator-app-integration.test.ts

```diff
@@ -6,21 +6,25 @@ import decache from "decache";
 import { PATH_DATA } from "../../../app.constants";
 import { UnsecuredJWT } from "jose";
 import { checkFailedCSRFValidationBehaviour } from "../../../../test/utils/behaviours";
-import * as mfaModule from "../../../utils/mfa";
 
 describe("Integration:: change authenticator app", () => {
   let token: string | string[];
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+
   let cookies: string;
-  const sandbox = sinon.createSandbox();
+  let sandbox: sinon.SinonSandbox;
+  let app: Awaited<ReturnType<typeof appWithMiddlewareSetup>>;
+
+  beforeEach(async () => {
+    sandbox = sinon.createSandbox();
```

Comment:
Definitely don't change it now, but we don't need to create a sandbox any more. Since Sinon v5 the base `sinon` object [is already a sandbox](https://sinonjs.org/releases/latest/sandbox/), so we can simplify this to 

```typescript
beforeEach(async () => {
    app = await appWithMiddlewareSetup();
});

afterEach(() => {
  sinon.restore();
});
```

Then everywhere we do eg. `sandbox.fake()` or `sandbox.stub()` we just use `sinon.stub()`.

Reply:
nice!